docs/developer/architecture/core/http-service.asciidoc

```diff
@@ -12,7 +12,7 @@ The service allows plugins to:
 * to execute custom logic on an incoming request or server response.
 * implement custom authentication and authorization strategy.
 
-See {kib-repo}blob/{branch}/docs/development/core/server/kibana-plugin-core-server.httpservicesetup.md[HTTP service API docs]
+See https://github.com/elastic/kibana/blob/main/packages/core/http/core-http-server/src/http_contract.ts[HTTP service contract types].
```

Comment:
Should we use the `{handlebar}` magic vars like below?
```suggestion
See {kib-repo}blob/{branch}/packages/core/http/core-http-server/src/http_contract.ts[HTTP service contract types].
```

Reply:
Your suggestion makes sense _if_ and only if we make sure that `{branch}` always points to a version > 8.4.0 when the [code was actually moved](https://github.com/elastic/kibana/pull/135562).
Since this PR targets 8.11, we should be safe to use handlebars.

Reply:
Hey @TinaHeiligers!

> Your suggestion makes sense if and only if we make sure that {branch} always points to a version > 8.4.0 when the https://github.com/elastic/kibana/pull/135562.

This should be the case. This variable is updated during each docs release to match the docs branch. Example: For the 8.8 docs, `{branch}` is `8.8`.  For the 8.9 docs, `{branch}` is 8.9.

> Is there some way we can pro-actively catch them, besides manually going through the docs?

Unfortunately not. The docs CI is set up to only check elastic.co/guide links ([source code](https://github.com/elastic/docs/blob/master/lib/ES/LinkCheck.pm)) so it won't check links to the Kibana repo or external links. There's an open issue to address this in Docsmobile, the new docs system: https://github.com/elastic/docsmobile/issues/460

I've opened an issue for our team to manually audit the links in the meantime: https://github.com/elastic/platform-docs-team/issues/165

Reply:
> I've opened an issue for our team to manually audit the GitHub links in the meantime

Thank you @jrodewig! I'l watch that issue. In the mean time, feel free to let the docs team know I'm ope. to questions.